creator-node/src/index.ts

```diff
@@ -100,6 +101,14 @@ const startApp = async () => {
     )
   }
 
+  try {
+    const solSigningAuthority = Keypair.fromSeed(privateKeyBuffer)
```

Comment:
we should attach this keypair to the app and simply serve the public key on demand instead of setting another config - this is on order to use it later

Reply:
what do you mean by this @hareeshnagaraj? once it's attached to the config, we can use it globally at any time.

Reply:
ah for sure in that case we should set the keypair in the config object if that is safe https://github.com/AudiusProject/audius-protocol/pull/2854/files#diff-fde27aec1d059a20ad60914bf50f20439230a767c4b362288d05aaa9e4b22a40R106

Reply:
got it. the whole key pair vs just the pubkey

Reply:
so it's safe to set the whole key pair (with the private key) on the app/config (whichever is decided)? I'm not sure of other programs that might be able to access that or if it gets shared anywhere else?

Reply:
forgot to tag @dmanjunath @hareeshnagaraj

Reply:
thanks, yeah that makes sense since the eth private key is already stored in the config so no added security risk there. will add the config.js documentation as well

Reply:
perfect thanks @theoilie. great question btw

Reply:
yes, definitely great question @theoilie !